data/available_datasets.json

```diff
@@ -1,6 +1,152 @@
 {
-    "last_updated": "2023-06-12",
+    "last_updated": "2023-07-31",
     "datasets": [
+        {
+            "doi": "10.1101/2023.07.05.23292245",
+            "available_items": [
+                {
+                    "type": "data",
+                    "repo_name": "GitHub",
+                    "accession_number": "",
+                    "description": "Data processing code in R is available on Github",
```

Comment:
description should be modified here to e.g. 'Data used in modelling, including location and population size'